test/e2e/framework/ginkgowrapper.go

```diff
@@ -544,8 +559,9 @@ func withFlaky() interface{} {
 type label struct {
 	// parts get concatenated with ":" to build the full label.
 	parts []string
-	// extra is an optional fully-formed extra label.
-	extra string
+	// extra is an optional feature name. It gets added as [<extraFeature>]
+	// to the test name and as Feature:<extraFeature> to the labels.
+	extraFeature string
```

Comment:
you got me puzzled with this, so there is only one change in this PR that is
```
	if slices.Contains(labels, "Feature:"+tag) {
				// Okay, was also set as label.
				continue
			}
```

the rest is a rename of this field?

Reply:
The only user-visible change is indeed that the `Alpha` Ginkgo label becomes `Feature:Alpha` and `Beta` becomes `Feature:Beta`.

I'm not aware of anyone currently using the Ginkgo labels, so I think it's fairly safe to do this.

I am not renaming the inline `[Alpha]` and `[Beta]` tags. Those have been around for longer, so they might be used?

Reply:
it was a side comment, this LGTM

Reply:
The alternative is to add *both* `[Alpha]` and `[FeatureGate:Alpha]` in the test text. Then we don't need to update jobs because `SKIP=\[FeatureGate:.+\]` will skip alpha tests.

I'm just mentioning this for the sake of completeness. I think we should go directly to the desired state instead of introducing new (temporary?) inline tags that users then might start to rely on.